### PR TITLE
Fix incorrect bool in gcloud.delete_instances.

### LINKED ISSUE #322: Experiment is not ended and dispatcher stays alive even at end of experiment.
Looks like measurer is not finished, scheduler did finish.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes #322

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Verified that this was the bug, aarya-test-9 exited successfully.

File: common/test_gcloud.py [hunk 1]
```diff
@@ -170,9 +170,9 @@ def test_delete_instances_greater_than_batch_size(mocked_execute):
   than batch size."""
     instances = ['instance-%d' % i for i in range(103)]
     mocked_execute.return_value = new_process.ProcessResult(0, '', False)
-    # -q is needed otherwise gcloud will prompt "Y/N?".
```
Comment (jonathanmetzman) [inline common/test_gcloud.py:173]: Why delete this?

File: common/gcloud.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def delete_instances(instance_names: List[str], zone: str, **kwargs) -> bool:
         result = new_process.execute(command, expect_zero=False, **kwargs)
         error_occurred = error_occurred or result.retcode != 0
 
-    return error_occurred
+    return not error_occurred
```
Comment (jonathanmetzman) [inline common/gcloud.py:134]: Can you update the docstring to say what return value is?

File: common/test_gcloud.py (hunk 1 above)
Comment (inferno-chromium) [reply common/test_gcloud.py:173]: It is there in source code in gcloud.delete_instances, not needed in test again.

File: common/gcloud.py (hunk 2 above)
Comment (inferno-chromium) [reply common/gcloud.py:135]: Done!

**Pull request merged**